docs/policy/collab-bearer-tokens.md

```diff
@@ -0,0 +1,134 @@
+Collaborations and Bearer Tokens
+================================
+
+Sites in the OSG grant access to their grid services based on client's association with a specific collaboration
+(i.e. VO) instead of granting access on a per-user basis.
+In the past, this type of access was provided through X.509 proxies with VOMS attributes to demonstrate assocation with
+a collaboration:
+
+-   Users (both human and robots) would request that their collaboration sign their X.509 proxies
+    (usually through a VOMS server),
+-   Or in the case of automated services (i.e. pilot job submission) a collaboration could directly create and sign
+    proxies themselves
+
+Now, sites can authenticate and authorize clients presenting bearer tokens, such as SciTokens or WLCG tokens.
+This document describes how collaborations can issue bearer tokens that are compatible with OSG sites.
+
+Issuers
+-------
+
+To generate bearer tokens, a collaboration must adminster at least one "token issuer" to issue tokens to their users.
+In addition to generating and signing tokens, token issuers provide a public endpoint that can be used to validate an
+issued token,
+e.g. an OSG Compute Entrypoint will contact the token issuer to authorize a bearer token used for pilot job submission.
+
+!!! attention "Token issuer uptime"
+    Due to the centralized nature of bearer token validation, token issuers should be treated as critical, highly
+    available services.
+    Otherwise, a token issuer outage will result in OSG sites being unable to authenticate a collaboration's tokens.
+
+Choose one of the token issuer types below, depending on the needs of your collaboration.
+
+### Simple issuer ###
+
+If your collaboration centrally administers all services requiring bearer tokens and your users do not need to directly
+manage bearer tokens, consider running a simple token issuer.
+A simple token issuer consists of a public/private certificate keypair where the private key is used to issue tokens
+directly and the public certificate is made available through a web server.
+
+For example, the OSPool (née OSG VO) serves its public certificate through
+[GitHub pages](https://github.com/scitokens/osg-connect/) and uses the private key to sign tokens used for pilot job
+submission as well as automatically generating tokens to accompany user jobs so that they can access their private
+storage areas.
```

Comment:
What's a good place to put the private key? How is the private key used to issue tokens? What commands do you need to run? Linking to someone else's documentation would be fine here.